src/helix-ui/icons/server.svg

```diff
@@ -1,3 +1,3 @@
 <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
-  <path d="M1 10.783h14v2.811C15 14.37 14.372 15 13.6 15H2.4c-.773 0-1.4-.63-1.4-1.406v-2.811zm0-.646V5.92h14v4.217H1zm14-4.92H1V2.406C1 1.63 1.628 1 2.4 1h11.2c.773 0 1.4.63 1.4 1.406v2.811zM3.107 13.398h3.36v-1.125h-3.36v1.125zm7.242-1.125a.56.56 0 0 0-.56.562.56.56 0 1 0 1.12 0 .56.56 0 0 0-.56-.562zm1.983 0a.56.56 0 0 0-.56.562.56.56 0 1 0 1.121 0 .561.561 0 0 0-.561-.562zM3.1 8.591h3.36V7.466H3.1v1.125zm7.249-1.125a.56.56 0 0 0-.56.562.561.561 0 1 0 1.12 0 .56.56 0 0 0-.56-.562zm1.983 0a.56.56 0 0 0-.56.562.561.561 0 1 0 1.121 0 .561.561 0 0 0-.561-.562zM3.1 3.728h3.36V2.603H3.1v1.125zm7.249-1.125a.56.56 0 0 0-.56.562.56.56 0 1 0 1.12 0 .56.56 0 0 0-.56-.562zm1.983 0a.56.56 0 0 0-.56.562.56.56 0 1 0 1.121 0 .561.561 0 0 0-.561-.562z"/>
+  <path fill-rule="evenodd" d="M1.5 11h13c.276 0 .5.246.5.546v2.909c0 .3-.224.545-.5.545h-13c-.275 0-.5-.245-.5-.545v-2.91c0-.3.225-.545.5-.545zm0-5h13c.276 0 .5.246.5.546v2.909c0 .3-.224.545-.5.545h-13c-.275 0-.5-.245-.5-.545v-2.91c0-.3.225-.545.5-.545zm0-5h13c.276 0 .5.246.5.546v2.909c0 .3-.224.545-.5.545h-13c-.275 0-.5-.245-.5-.545v-2.91c0-.3.225-.545.5-.545zM11 13.032c.022.534.493.98 1.02.968.533-.012.983-.474.98-1.007a1.016 1.016 0 0 0-1.046-.992A1.02 1.02 0 0 0 11 13.032zm0-5c.022.534.493.98 1.02.968.533-.012.983-.474.98-1.007a1.016 1.016 0 0 0-1.046-.992A1.02 1.02 0 0 0 11 8.032zm0-5c.022.534.492.979 1.02.967A1.02 1.02 0 0 0 13 2.993a1.015 1.015 0 0 0-1.046-.992A1.02 1.02 0 0 0 11 3.032zM3 14h6v-2H3v2zm0-5h6V7H3v2zm0-5h6V2H3v2z"/>
```

Comment:
🎨 Previous icon design was a remnant of the dark days. Bringing up to speed with modern times.